mods/flowers/init.lua

```diff
@@ -101,15 +160,15 @@ for _, m in pairs(mushrooms_datas) do
 		}
 	})
 
-	-- Register spores
+	-- Register mushroom spores
 
-	minetest.register_node("flowers:spores_" .. name, {
+	minetest.register_node("flowers:mushroom_spores_" .. name, {
 		description = string.sub(string.upper(name), 0, 1) ..
 			string.sub(name, 2) .. " Mushroom Spores",
```

Comment:
As I'm dishonestly accusated to do not test my suggestions... Here's some evidence : 

![2015-08-07-202656_1280x800_scrot](https://cloud.githubusercontent.com/assets/7883281/9142925/349ea908-3d43-11e5-8b09-071c9ae96610.png)

![2015-08-07-202659_1280x800_scrot](https://cloud.githubusercontent.com/assets/7883281/9142930/39f3d392-3d43-11e5-93c6-e81e1c5f6a7b.png)